GCRCatalogs/dc2_truth_match.py

```diff
@@ -0,0 +1,87 @@
+"""
+Reader for truth-match catalogs persisted as parquet files and partitioned in tracts.
+"""
+import numpy as np
+import astropy.units as u
+
+from .dc2_dm_catalog import DC2DMTractCatalog
+
+__all__ = ["DC2TruthMatchCatalog"]
+
+
+def _flux_to_mag(flux):
+    with np.errstate(divide="ignore"):
+        return (flux * u.nJy).to_value(u.ABmag)  # pylint: disable=no-member
+
+
+class DC2TruthMatchCatalog(DC2DMTractCatalog):
+    r"""
+    DC2 Truth-Match (parquet) Catalog reader
+
+    Presents tables exactly as they are defined in the files (no aliases,
```

Comment:
What about the added magnitude columns?
And, for addon case, the name changes to avoid collisions with names in object?